packages/components/src/__future__/Tag/RemovableTag/_docs/RemovableTag.stickersheet.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { RemovableTag } from "../index"
+
+export default {
+  title: "Components/Tag/Future/RemovableTag",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: () => (
+    <>
+      <StickerSheet heading="Button Group">
+        <StickerSheet.Header headings={["Default", "Hover", "Focus visible"]} />
+        <StickerSheet.Body>
+          <StickerSheet.Row>
+            <RemovableTag
+              removeButtonProps={{
+                ariaLabel: "Remove tag",
+                onClick: () => alert("clicked"),
+              }}
+            >
+              My tag
+            </RemovableTag>
+            <RemovableTag
+              data-sb-pseudo-styles="hover"
+              removeButtonProps={{
+                ariaLabel: "Remove tag",
+                onClick: () => alert("clicked"),
+              }}
+            >
+              My tag
+            </RemovableTag>
+            <RemovableTag
+              data-sb-pseudo-styles="focus"
+              removeButtonProps={{
+                ariaLabel: "Remove tag",
+                onClick: () => alert("clicked"),
+              }}
+            >
+              My tag
+            </RemovableTag>
+          </StickerSheet.Row>
+        </StickerSheet.Body>
+      </StickerSheet>
+    </>
+  ),
+  parameters: {
+    pseudo: {
+      hover: [
+        '[data-sb-pseudo-styles="hover"]',
+        '[data-sb-pseudo-styles="hover"] > button',
+      ],
+      focusVisible: [
+        '[data-sb-pseudo-styles="focus"]',
+        '[data-sb-pseudo-styles="focus"] > button',
+      ],
```

Comment:
You'll need a `focus` version of this too

Reply:
Can do. Do we want `:focus` and `:focus-visible` states to have the same styles?

Reply:
Yeah, same config just need a `focus` key